data_steward/utils/slack_alerts.py

```diff
@@ -62,6 +98,13 @@ def _get_slack_client():
     return slack.WebClient(slack_token)
 
 
+def check_channel_and_token():
+    if _is_channel_available and _is_token_valid:
+        return True
+    else:
+        return False
+
+
```

Comment:
This function could potentially go.

Reply:
I removed it because I removed the `is_token_valid` function and we will be calling `is_channel_available` directly in the `slack_logging_handler` module.